java/ql/test/query-tests/Metrics/GeneratedVsManualCoverage/IOUtils.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.commons.io;
+import java.io.*;
+import java.util.*;
+
+public class IOUtils {
+  public static BufferedInputStream buffer(InputStream inputStream) { return null; } // Generated-only summary
```

Comment:
It would probably be nice to group these methods and separate the groups with an additional newline to make it a bit more readable.

Reply:
Updated in https://github.com/github/codeql/pull/11585/commits/028fc2963933fcf396b434a3e243ccf89518062f.